hooks/useEasyWrite.ts

```diff
@@ -96,6 +97,10 @@ export const useEasyWrite = (params: UsePrepareContractWriteConfig) => {
     config.request?.chainId,
   ]);
 
+  useEffect(() => {
+    if (writeError) reset();
+  }, [params]);
```

Comment:
This works, and if I am understanding correctly a simpler solution may be to remove the error conditional in [here](https://github.com/ScopeLift/l2-flexible-voting-frontend/blob/main/pages/bridge.tsx#L235)

Reply:
You're absolutely right, thank you!!

Reply:
implemented: [e0d36ac](https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/108/commits/e0d36ac81a82806849278110712ee54aab0eb266)